training/src/app/main/main.component.scss

```diff
@@ -1,128 +1,116 @@
 @import "src/styles.scss";
 
+.main {
+  display: flex;
+  flex-direction: column;
+}
 .main-pic{
   position: relative;
-  height: fit-content;
-  align-items: center;
-  width: 100%;
   display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 70px;
 }
+
 #fire {
   position: relative;
   width: 100%;
   height: 100%;
 }
 #chuck-pic {
   position: absolute;
-  width: 24%;
-  height: 55%;
-  margin-left: 38%;
-  margin-top: 40%;
+  max-width: 20%;
+  max-height: 40%;
   filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.11));
   border-radius: 142.5px;
-}
-#chuck-text{
-  position: absolute;
-  width: 28.8%;
-  height: 14.5%;
-  margin-left: 35.8%;
-  margin-top: 51%;
-}
-.parent {
-  position: relative;
-  width: fit-content;
-  block-size: fit-content;
-  font-size: 14px;
+  transform: translateY(100%);
+  margin-bottom: 15px;
 }
 
 .rectangle-1{
-  width: 60%;
+  position: relative;
+  width: 912px;
```

Comment:
we said its kind of bad to give this large width.
what happen for users with screen of 700px? it will be outside of limitation.
you can completely remove the width, so it will automatic take the full width and be responsive.
and plus, u can add  padding: 0 10px; so the inside elements wont be so close to the sides